code/controllers/subsystem/shuttle.dm

```diff
@@ -221,7 +221,7 @@ SUBSYSTEM_DEF(shuttle)
 /datum/controller/subsystem/shuttle/proc/centcom_recall(old_timer, admiral_message)
 	if(emergency.mode != SHUTTLE_CALL || emergency.timer != old_timer)
 		return
-	emergency.cancel(/area/centcom)
+	emergency.cancel(new /area/centcom)
```

Comment:
@Cyberboss there is no actual centcom area. There's only the children. I checked when debugging this.